packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -327,11 +330,30 @@ function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Pr
     device: device!,
   });
 
+  const workspace = useWorkspaceConfig();
+
+  const isFrameDisabled = workspace.showDeviceFrame === false;
+
+  function MaybeFrame() {
```

Comment:
Please just name it `DeviceFrame` and move `workspace` & `isFrameDisabled` inside.

Reply:
@jakub-gonet `cssPropertiesForDevice` takes `isFrameDisabled`  because styles depend on it. So I cannot move these to `DeviceFrame` component